app/views/users/_user.html.haml

```diff
@@ -8,6 +8,6 @@
   %td
     = date_mmddyyyy(user.created_at)
   %td
-    = time_ago_in_words(user.current_sign_in_at)
+    = time_ago_in_words(user.current_sign_in_at || user.created_at)
```

Comment:
should there be a sort test for this change?

Reply:
to be addressed in #589